mqtt/mqtt-bridge/src/persist/mod.rs

```diff
@@ -45,6 +47,13 @@ pub enum PersistError {
 
     #[error("Cannot remove key {current} that is not in order, but {expected} expected")]
     BadKeyOrdering { current: Key, expected: Key },
+
+    #[error("Unexpected loader state. key={key}, loaded={loaded:?}, new_batch={new_batch:?}")]
+    Loader {
+        key: Key,
+        loaded: Vec<Key>,
+        new_batch: Vec<(Key, Publication)>,
+    },
```

Comment:
From my comment elsewhere, I think just the keys should be good enough right? I, personally, don't like the possibility of putting publication content in the logs and think we should avoid doing that whenever possible.

Reply:
well, sometimes they are useful.. probably you are right